droidmorphing-lib/build.gradle

```diff
@@ -1,25 +1,23 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 19
-    buildToolsVersion "20.0.0"
+    compileSdkVersion 21
+    buildToolsVersion "21.1.2"
 
     defaultConfig {
-        applicationId "com.alexkolpa.droidmorphing"
```

Comment:
not allowed, actually. applicationIds for libs need to be placed in the AndroidManifest for the new gradle plugin versions